spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,58 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'with an unauthenticated user' do
+      let(:params) { attributes_for :conference }
+
+      it 'cannot create new conference' do
+        expect {
+          post :create,
+          params: { conference: params }
+        }.not_to change { Conference.count }
+      end
+
+      it 'cannot create new conference' do
+        expect {
+          post :create,
+          params: { conference: params },
+          xhr: true
+        }.not_to change { Conference.count }
+      end
+
+      it 'should return a 401 response status' do
+        post :create, params: { conference: params }, xhr: true
+        expect(response).to have_http_status(401)
+      end
+
+    end
+
+    context 'with user logged in' do
+      let(:student) { FactoryGirl.create(:student) }
+      let(:params) { attributes_for :conference }
```

Comment:
Same here for naming: these aren't the "params". Maybe `conference_attrs` would be a better choice, or moving it into something like:
```ruby
let(:params) { { conference: attributes_for(:conference) } }
```